test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/NodeTests.cs

```diff
@@ -31,16 +35,17 @@ namespace Microsoft.SqlTools.ServiceLayer.UnitTests.ObjectExplorer
     public class NodeTests : ObjectExplorerTestBase

```

Comment:
Is it possible to separate tests for this project in a separate project too? e.g. if new features are added to this project ONLY, you won't need to add tests to STS but to this projects' test suite.